init_scripts/init-root.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+echo "nameserver 8.8.8.8" | sudo tee /etc/resolv.conf
+# Stay up-to-date
+apt-get -y update
```

Comment:
This would make your deveopment environment not reproducible. You will get a different environment every time you run the Vagrant deployment. Please avoid doing that.